### PR TITLE
fix code quality issues

### PR DESCRIPTION
**Pull request by Nelwell**

### CODE REVIEW

**Comment by claraj** [comment]
@John151 can you review?

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,66 @@
...
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    Read in JSON data and loop over
+    """
+    with open('data.json', 'r') as file:
+        state_names = file.read()
+        state_names = json.loads(state_names)
+    print(state_names)
```
Comment (John151) [inline convert_state.py]: Print statement isn't necessary

**Review by John151 — COMMENTED** [review]
Looks really good! 1 print statement note. Changing output to f string was a good idea!

File: convert_state.py (hunk 1 above)
Comment (Nelwell) [reply convert_state.py]: good call.  Not good for the user exp.

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,65 @@
...
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    Read in JSON data and loop over
+    """
+    with open('data.json', 'r') as file:
+        state_names = file.read()
```
Comment (claraj) [inline convert_state.py:13]: state_names is a string on this line

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,65 @@
...
+        state_names = json.loads(state_names)
```
Comment (claraj) [inline convert_state.py]: Then state_names is overwritten with a dictionary.  Bad style to change the type of a variable. Languages like Java will prevent this, Python assumes you know what you are doing. This code works but it's bad style and confusing as to what the actual type of state_names is

**Review by claraj — CHANGES_REQUESTED** [review]
Looks so much better! The  re-use of the state_names variable is the only issue to look at.

File: convert_state.py (hunk 3 above)
Comment (Nelwell) [reply convert_state.py]: Made the change and pushed. Thanks! Didn't catch this in class.

**Pull request closed without merging**